huaweicloud/services/hss/resource_huaweicloud_hss_host_group.go

```diff
@@ -362,3 +368,16 @@ func resourceHostGroupDelete(_ context.Context, d *schema.ResourceData, meta int
 
 	return nil
 }
+
+func resourceHostGroupImportState(_ context.Context, d *schema.ResourceData, _ interface{}) (
+	[]*schema.ResourceData, error) {
+	parts := strings.Split(d.Id(), "/")
+	if len(parts) != 2 {
+		return nil, fmt.Errorf("invalid format of import ID, must be <enterprise_project_id>/<id>")
+	}
+
+	d.Set("enterprise_project_id", parts[0])
+	d.SetId(parts[1])
+
+	return []*schema.ResourceData{d}, nil
```

Comment:
Please handle the error from `d.Set("enterprise_project_id", parts[0])`
```suggestion
	return []*schema.ResourceData{d}, d.Set("enterprise_project_id", parts[0])
```

Reply:
fixed.